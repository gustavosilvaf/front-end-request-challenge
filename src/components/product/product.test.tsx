import { render } from "@testing-library/react";
import { Product } from ".";

describe("Test product render", () => {
  const mockedProductName = "Fake Shirt";

  it("Should render product corretly", () => {
    const { asFragment } = render(<Product name={mockedProductName} />);
    expect(asFragment).toMatchSnapshot();
  });

  it("Should find name that was passed", () => {
    const { getByText } = render(<Product name={mockedProductName} />);

    expect(getByText(mockedProductName)).toBeInTheDocument();
  });
});
