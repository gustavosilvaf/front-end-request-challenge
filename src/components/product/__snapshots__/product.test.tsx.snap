// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test product render Should render product corretly 1`] = `[Function]`;
